### DISCUSSION TITLE
react native expo

### DISCUSSION THREAD

**Discussion by Alice362** [community/community#46721] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

i have been trying to setup my expo app but i get this warning
**WARNING: The legacy expo-cli does not support Node +17. Migrate to the versioned Expo CLI (npx expo).**
should get worried about it???

**Comment by jge162** [comment]
No, you don't need to worry about it @Alice362 It is just letting you expo-CLI does not support Node +17. Migrate -> meaning you should probably start using Expo CLI (nxp expo).

**Comment by Edleychris** [comment]
try downgrading the node version to 16 run this on your terminal

**Comment by AbeDev404** [comment]
You don't need to worry about it. This warning simply means that the version of expo-cli you are using does not support Node.js version 17 or higher. To fix this, you can simply switch to the new versioned expo-cli by running npx expo instead of expo. The new versioned expo-cli should work with Node.js version 17 or higher.
